python/tensorflowjs/converters/keras_h5_conversion.py

```diff
@@ -127,6 +129,33 @@ def _ensure_json_dict(item):
   return item if isinstance(item, dict) else json.loads(as_text(item))
 
 
+# https://github.com/tensorflow/tfjs/issues/1255, b/124791387
+# In tensorflow version 1.13 and some alpha and nightly-preview versions,
+# the following layers have different class names in their serialization.
+# This issue should be fixed in later releases. But we include the logic
+# to translate them anyway, for users who use those versions of tensorflow.
+_CLASS_NAME_MAP = {
+    'BatchNormalizationV1': 'BatchNormalization',
+    'UnifiedGRU': 'GRU',
+    'UnifiedLSTM': 'LSTM'
+}
+
+
+def translate_class_names(input_object):
+  if not isinstance(input_object, dict):
+    return input_object
+  out = copy.copy(input_object)
```

Comment:
can you replace the name inline instead of copying?

Reply:
I was thinking we should be save and not modify the input in place. But since the arg is not used elsewhere, modifying it in place should be find. I removed the copy.copy. Thanks.
PTAL